opt_adapt/opt.py

```diff
@@ -171,8 +243,13 @@ def minimise(
         dJ_ = None if it == 1 else op.dJdm_progress[-1]
         if order == 1:
             args = (u_plus, u_, dJ_)
+        if method == "BFGS":
+            B = None
+            args = (u_plus, u_, dJ_, B)
+        if method == "newton_method":
+            args = (u_plus)
```

Comment:
Can you not write it so that it says
```elif order == 2:```
and have both BFGS and Newton having the same interface?